tools/make/data.toml

```diff
@@ -133,19 +133,35 @@ args = [
 ]
 
 [tasks.full-data]
-description = "Builds a complete postcard file from latest CLDR and icu_exportdata."
+description = "Builds full baked data and verifies that all locales generate correctly"
```

Comment:
Nit: I think the globaldata should be in its own job that could be run in the same CI as full-data, and it should have a diff-checker variant

Reply:
Once we have data in all crates, this will be the same as the full-data check. We can then change the last line to `--locales basic moderate`, which will be super fast because the binary is already built.

Reply:
Ah gotcha